op-chain-ops/genesis/config.go

```diff
@@ -682,10 +680,14 @@ func NewL2ImmutableConfig(config *DeployConfig, block *types.Block) (immutables.
 
 	immutable["L2StandardBridge"] = immutables.ImmutableValues{
 		"otherBridge": config.L1StandardBridgeProxy,
+		"messenger":   predeploys.L2CrossDomainMessengerAddr,
 	}
 	immutable["L2CrossDomainMessenger"] = immutables.ImmutableValues{
 		"otherMessenger": config.L1CrossDomainMessengerProxy,
 	}
+	immutable["OptimismMintableERC20Factory"] = immutables.ImmutableValues{
```

Comment:
Any reason to move this from the bottom? It just increases the diff

Reply:
The diff at the bottom removes the `storage` entry, this is adding one to `immutable`.

Reply:
I moved this entry to the end of the immutable config just for good measure though.